### ISSUE TITLE
Each segment should start on the next minute

### ISSUE THREAD

**Issue by Matt-Deacalion** [Matt-Deacalion/Pomodoro-Calculator#10]
At the moment, where one segment ends is where the next one picks up. It's on exactly the same minute. This should flow over to the next minute.

**Comment by Matt-Deacalion** [comment]
Fixed in a01704de6cc524ec1b19e5df223eff079b85fb8c.

**Comment by freayd** [comment]
Hi Matt! It think that the previous version was actually correct. If I wanted to work between 8:00 and 10:00, I should have 4 pomodori:
- [8:00-8:25[ (work including the 8:00 minute, but excluding the 8:25 minute)
- [8:30-8-55[
- [9:00-9:25[
- [9:30-9:55[
When you say "At the moment, where one segment ends is where the next one picks up", I think this is how time segments should behave. If I work from 8:00 to 8:25, it means that I stop working at 8:25:00, and my break start at 8:25:00. If the calculator tell me to start the break at 8:26:00, it means that between 8:25:00 and 8:26:00 I wasn't working nor in break.

Here's an what the program currently calculates:
```
$ get-pomodori --from=8:00 10:00
 1 Pomodoro     08:00 ⇾ 08:25
   short break  08:26 ⇾ 08:31
 2 Pomodoro     08:32 ⇾ 08:57
   short break  08:58 ⇾ 09:03
 3 Pomodoro     09:04 ⇾ 09:29

   Total Pomodori: 3
       Total Work: 1 hour, 15 minutes
```
But I think I should be able to do 4 pomodori in 2 hours.
2 hours = 120 minutes = 25 + 5 + 25 + 5 + 25 + 5 + 25 + 5 = 4 pomodori + 5 spare minutes at the end

**Comment by Matt-Deacalion** [comment]
I think you're right. I will change it back in the next few days. Thank you!

**Comment by freayd** [comment]
Thanks!

**Comment by domenn** [comment]
I am checking the commit that made this change. I propose we configure "transition period" cmd parameter, which will default to 0. So running with default parameter will cause behaviour like before a01704d. If we set for example transition--period 00:01:00 (or some other format of value), we would get current behaviour.

**Comment by Matt-Deacalion** [comment]
@domenn @freayd Thank you for being so patient.

Fixed in ba69c1a03b103ae24e5bc6ff05d83f9d9c47cf2d, released in fd426f479748826b4987f9037483686da5e344ec. Now on PyPi.

**Comment by freayd** [comment]
Wonderful, thanks a lot!
